02-intermediate-front-end/css2/readme.md

```diff
@@ -24,54 +24,6 @@ Because you have to start somewhere.
 
 Resize.  Resize.  Resize.
 
-## [display](http://learnlayout.com/display.html)
-
-`display` is CSS's most important property for controlling layout.
-
-Stomp. Stomp.  Stomp.
-
-I repeat... `display` is CSS's **most important property** for controlling layout.
-
-[The Usual Suspects](https://en.wikipedia.org/wiki/The_Usual_Suspects):
-- block
-- inline
-- none
-
-Demo (compare visibility/display): http://codepen.io/mattscilipoti/pen/JdpdoJ
-
-To give you an idea of just how big the world is that you are entering, check out this list of [display values](https://developer.mozilla.org/en-US/docs/CSS/display).  
-
-Don't worry about learning it all.  Focus on learning how to identify your specific problem and research the tools to answer it.
-
-## [margin: auto;](http://learnlayout.com/margin-auto.html)
-
-Useful for... centering.  Kinda.
-   - use `text-align:center` for `inline` and `inline-block`
-   - `margin:0 auto` for `block`
-
-## [max-width](http://learnlayout.com/max-width.html)
-
-And its twin: `min-width`
-
-## [The Box Model](http://learnlayout.com/box-model.html)
-
-You Do: One of these things is not like the other...
-> Compare the top & bottom box.  
-> 1. Are they the same width?
-> 2. What is different?
-> 3. What settings would you use to make them equal?
-
----
-
-Answers:
-1. Yes.  `width: 500px;`
-2. `padding` and `border`
-3. `.fancy { width: 386 }`
-
-## [box-sizing](http://learnlayout.com/box-sizing.html)
-
-`box-sizing: border-box;` - just one solution to this problem.
-
```

Comment:
Could replace the removed content with some sort of student-driven review (e.g., turn-and-talk) of the CSS properties they know that control layout.